Application/EdFi.Ods.WebApi/appsettings.json

```diff
@@ -75,7 +75,7 @@
     "ExcludedExtensions": []
   },
   "Plugin": {
-    "Folder": "",
+    "Folder": "../../Plugin",
```

Comment:
@raghuramakrishna plugin folder setting change is not needed for this ticket

Reply:
Updated